src/constants/ApiRoutes.js

```diff
@@ -11,6 +11,10 @@ const apiRoutes = {
     watchlist: {
       get: (accountId) => `/account/${accountId}/watchlist/movies`,
       toggle: (accountId) => `/account/${accountId}/watchlist`
+    },
```

Comment:
generally we call file starting with upercase letter if it is class or component